docs/user/installation_and_configuration.rst

```diff
@@ -132,26 +132,37 @@ For Accumulo 1.6+
 
 Copying the runtime JAR to each tablet server as for Accumulo 1.5 above will
 still work, but in Accumulo 1.6, we can leverage namespaces to isolate the
-GeoMesa classpath from the rest of Accumulo. First, you have to create the
-namespace in the Accumulo shell:
+GeoMesa classpath from the rest of Accumulo.
+
+Execute the ``install-geomesa-namespace.sh`` script in the ``geomesa-$VERSION/dist/accumulo`` directory.
 
 .. code::
 
+    $ ./install-geomesa-namespace.sh -u root -n myNamespace
```

Comment:
does the user have to be root? have we tested with anything else? since it grants permissions, a regular user maybe wouldn't work?
if it has to be root, we probably need 2 sets of users - the root user/pwd and then the runtime user that will be granted the permissions.